_pages/download.md

```diff
@@ -24,7 +24,14 @@ Here you can find the latest development snapshot(s). This versions have not bee
 
 Use them on your own risk and make backups of your maps in order to avoid trouble.
 
-<a class="btn btn-lg btn-primary" href="http://ci.safranil.fr/job/ElectricalAge/lastSuccessfulBuild" role="button">Download development snapshots</a>
+*For 1.12 development, there are currently no available jars. You will need to go to the following link and compile it yourself.*
+
+<a class="btn btn-lg btn-primary" href="https://github.com/Electrical-Age/ElectricalAge/tree/ports/1.10" role="button">1.12 Development Repository</a>
```

Comment:
The link points to the 1.10 port branch and text says 1.12. Is that correct?

Reply:
Yes, that's correct, from my understanding. @Baughn started that branch thinking we were doing 1.10 first, but from my understanding, he's just going straight for 1.12 because there's no reason to do 1.10 now. He didn't change the name when he did that.

Reply:
Alternatively, we could just link directly to the top level of the GitHub repository. I just figured it would be convenient to direct them to the right source tree.